src/allmydata/storage/http_client.py

```diff
@@ -355,6 +355,31 @@ def get_version(self):
         decoded_response = yield _decode_cbor(response, _SCHEMAS["get_version"])
         returnValue(decoded_response)
 
+    @inlineCallbacks
+    def add_or_renew_lease(
+        self, storage_index: bytes, renew_secret: bytes, cancel_secret: bytes
+    ):
```

Comment:
Maybe a return type annotation too?

Probably we should adopt some policy regarding annotations project-wide...

(Ah, I see this is just moved code, so this change is certainly optional)